hack/make/.build-rpm/docker-engine.spec

```diff
@@ -0,0 +1,168 @@
+
+# copy udev rules
+mkdir -p $RPM_BUILD_ROOT/etc/udev/rules.d
+cp -aT contrib/udev/80-docker.rules $RPM_BUILD_ROOT/etc/udev/rules.d/80-docker.rules
+
+# add init scripts
+mkdir -p $RPM_BUILD_ROOT/%{_unitdir}
+mkdir -p $RPM_BUILD_ROOT/etc/sysconfig
+mkdir -p $RPM_BUILD_ROOT/etc/init.d
+
+cp -aT contrib/init/systemd/docker.service $RPM_BUILD_ROOT/%{_unitdir}/docker.service
+cp -aT contrib/init/systemd/docker.socket $RPM_BUILD_ROOT/%{_unitdir}/docker.socket
+
+cp -aT contrib/init/sysvinit-redhat/docker.sysconfig $RPM_BUILD_ROOT/etc/sysconfig/docker
+cp -aT contrib/init/sysvinit-redhat/docker $RPM_BUILD_ROOT/etc/init.d/docker
+
+# add bash completions
+mkdir -p $RPM_BUILD_ROOT/usr/share/bash-completion/completions
+mkdir -p $RPM_BUILD_ROOT/usr/share/zsh/vendor-completions
+mkdir -p $RPM_BUILD_ROOT/usr/share/fish/completions
+cp -aT contrib/completion/bash/docker $RPM_BUILD_ROOT/usr/share/bash-completion/completions/docker
+cp -aT contrib/completion/zsh/_docker $RPM_BUILD_ROOT/usr/share/zsh/vendor-completions/_docker
+cp -aT contrib/completion/fish/docker.fish $RPM_BUILD_ROOT/usr/share/fish/completions/docker.fish
+
+# add man pages
+mkdir -p $RPM_BUILD_ROOT/%{_mandir}/man1
+cp -ar docs/man/man1/* $RPM_BUILD_ROOT/%{_mandir}/man1/
+
+# add vimfiles
+mkdir -p $RPM_BUILD_ROOT/usr/share/vim/vimfiles/doc
+mkdir -p $RPM_BUILD_ROOT/usr/share/vim/vimfiles/ftdetect
+mkdir -p $RPM_BUILD_ROOT/usr/share/vim/vimfiles/syntax
+cp -aT contrib/syntax/vim/doc/dockerfile.txt $RPM_BUILD_ROOT/usr/share/vim/vimfiles/doc/dockerfile.txt
+cp -aT contrib/syntax/vim/ftdetect/dockerfile.vim $RPM_BUILD_ROOT/usr/share/vim/vimfiles/ftdetect/dockerfile.vim
+cp -aT contrib/syntax/vim/syntax/dockerfile.vim $RPM_BUILD_ROOT/usr/share/vim/vimfiles/syntax/dockerfile.vim
+
+
+# list files owned by the package here
+%files
+%{_bindir}/docker
+/usr/libexec/docker/dockerinit
+/etc/udev/rules.d/80-docker.rules
+%{_unitdir}/docker.service
+%{_unitdir}/docker.socket
+/etc/sysconfig/docker
+/etc/init.d/docker
+/usr/share/bash-completion/completions/docker
+/usr/share/zsh/vendor-completions/_docker
+/usr/share/fish/completions/docker.fish
+%doc
+%{_mandir}/man1/*
+/usr/share/vim/vimfiles/doc/dockerfile.txt
+/usr/share/vim/vimfiles/ftdetect/dockerfile.vim
+/usr/share/vim/vimfiles/syntax/dockerfile.vim
+
+%post
+%if 0%{?is_systemd}
+%systemd_post docker
+%else
+# This adds the proper /etc/rc*.d links for the script
+/sbin/chkconfig --add docker
+%endif
+if ! getent group docker > /dev/null; then
+    groupadd --system docker
+fi
+
+%preun
+%if 0%{?is_systemd}
+%butts_preun docker
+%else
+if [ $1 -eq 0 ] ; then
+    /sbin/service docker stop >/dev/null 2>&1
+    /sbin/chkconfig --del docker
+fi
+%endif
+
+%postun
+%if 0%{?is_systemd}
+%butts_postun_with_restart docker
+%else
+if [ "$1" -ge "1" ] ; then
+    /sbin/service docker condrestart >/dev/null 2>&1 || :
+fi
+%endif
+
+%changelog
+* Fri May 01 2015 Jessie Frazelle <jess@docker.com> 1.7.0-1
+* - Initial rpm package
```

Comment:
I wonder if we could generate this block sanely, because there's no way we'll keep it up to date.  Is it installed inside the package like Debian does?